src/EditorFeatures/CSharpTest/QuickInfo/SemanticQuickInfoSourceTests.cs

```diff
@@ -7271,6 +7271,35 @@ public interface ICloneable<T>
                 Documentation("Clones a Test<T>."));
         }
 
+        [Fact]
+        public async Task TestInheritdocWithTypeParamRef1()
+        {
+            var markup =
+@"
+public interface ITest
+{
+    /// <summary>
+    /// A generic method <typeparamref name=""T""/>.
+    /// </summary>
+    /// <typeparam name=""T"">A generic type.</typeparam>
+    void Foo<T>();
+}
+
+public class Test : ITest
+{
+    /// <inheritdoc/>
+    public void $$Foo<T>() { }
+}";
+
+            await TestWithOptionsAsync(TestOptions.Regular8,
+                markup,
+                MainDescription($"void Test.Foo<T>()"),
+                Documentation("A generic method T."),
```

Comment:
❓ What did this say prior to the change?

Reply:
The documentation text prior were the same, but the tags types were `[text, space, text, text]` instead of `[text, space, typeparameter, text]`.

Prior to this change, `GetDocumentationComment` returns this XML:

```xml
A generic method <see cref="?:T">
```

which fools the caller in thinking `T` is a bad `cref`.

With, this change, the XML becomes:

```xml
A generic method <typeparamref name="T">
```